openshift/scheduler.yaml

```diff
@@ -0,0 +1,271 @@
+- kind: DeploymentConfig
+  apiVersion: v1
+  metadata:
+    annotations:
+      description: Defines how to deploy the celery scheduler
+      template.alpha.openshift.io/wait-for-ready: "true"
+    name: ${NAME}-scheduler
+    labels:
+      app: ${NAME}
+  spec:
+    replicas: 1
+    selector:
+      name: ${NAME}-scheduler
+    resources:
+      requests:
+        cpu: ${CELERY_SCHEDULER_CPU_REQUEST}
+        memory: ${CELERY_SCHEDULER_MEMORY_REQUEST}
+      limits:
+        cpu: ${CELERY_SCHEDULER_CPU_LIMIT}
+        memory: ${CELERY_SCHEDULER_MEMORY_LIMIT}
+    strategy:
+      type: Recreate
+    template:
+      metadata:
+        labels:
+          name: ${NAME}-scheduler
+        name: ${NAME}-scheduler
+      spec:
+        containers:
+        - name: ${NAME}-scheduler
+          image: ${NAME}-scheduler
+          command:
+            - /bin/bash
+            - -c
+            - >
+              PYTHONPATH=${PWD}/rbac/ scl enable rh-python36 -- celery -A
+              rbac.celery beat -l $DJANGO_LOG_LEVEL
+          env:
+            - name: REDIS_HOST
+              valueFrom:
+                configMapKeyRef:
+                  name: redis-config
+                  key: redis-service-host
+                  optional: false
+            - name: REDIS_PORT
+              valueFrom:
+                configMapKeyRef:
+                  name: redis-config
+                  key: redis-service-port
+                  optional: false
+            - name: DJANGO_LOG_LEVEL
+              valueFrom:
+                configMapKeyRef:
+                  name: rbac-env
+                  key: django-log-level
+                  optional: false
+            - name: DJANGO_DEBUG
+              valueFrom:
+                configMapKeyRef:
+                  name: rbac-env
+                  key: django-debug
+                  optional: true
+            - name: APP_NAMESPACE
+              valueFrom:
+                configMapKeyRef:
+                  name: rbac-env
+                  key: app-namespace
+                  optional: true
+            - name: CW_AWS_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  key: cw-aws-access-key-id
+                  name: cloudwatch
+                  optional: true
+            - name: CW_AWS_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: cw-aws-secret-access-key
+                  name: cloudwatch
+                  optional: true
+            - name: CW_AWS_REGION
+              valueFrom:
+                secretKeyRef:
+                  key: cw-aws-region
+                  name: cloudwatch
+                  optional: true
+            - name: CW_LOG_GROUP
+              valueFrom:
+                secretKeyRef:
+                  key: cw-log-group
+                  name: cloudwatch
+                  optional: true
+            - name: ROLE_SEEDING_ENABLED
+              value: "False"
```

Comment:
In Django settings you have this defaulting to True; same in worker.yaml

Reply:
Intended.